tools/importers/SynchronizeTypographyTokens.spec.ts

```diff
@@ -0,0 +1,115 @@
+import {
+	ImportTypographyVariables,
+	ImportTypographyTokens,
+	requestForBin,
+} from './SynchronizeTypographyTokens';
+const fs = require('fs');
+const tokensFilesConfig = require('./configs/FakeSynchronizeTypographyTokensConfig.json');
+
+describe('Typography Tokens', () => {
+	it('check config & create files', async () => {
+		let requestResponse = await requestForBin(
```

Comment:
do we request external service in tests?
It isn't a good practice. What if UX updates JSON file, so we get different items than we expect?

Reply:
I know it's not the best practice. But if something will be broken in the JSON files from Figma.

> What if UX updates JSON file, so we get different items than we expect?

Hmm we should know about it, but it true it shouldn't break CI at a random moment. I will convert it to local files and we will get rid of other problems.

Reply:
Fixed, we are using now static file, which has minimum structure.